Hush/Assets/Scripts/Weapon/PlayerSword.cs

```diff
@@ -32,8 +32,8 @@ public class PlayerSword : MonoBehaviour, IWeapon
 
         public WeaponType WeaponType => WeaponType.Sword;
     
-        public float BaseDamage => baseDamage;
-        public float HeavyDamage => heavyDamage;
+        public float BaseDamage => baseDamage + GameManager.Instance.GetPlayerDamageBoost();
```

Comment:
Damage boost doesn't affect fireball damage ?

Reply:
No, because its listed as a strength boost in the menu. I should rename the variable tbh